src/main/resources/db/migration/mysql/V1.7.0__add-clear-duplicate-key.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE IF NOT EXISTS duplicates_history (
```

Comment:
Can the logs be added to identify how many of the duplicate records been removed? This will help identify the volumes.

Reply:
If we do a `select count(1) from duplicates_history` that will identify how many times a user had encountered the duplicate token problem. As the duplicate issue occurs on creation of the second token and prevents another login attempt, this means users should not be able to get a third token, so multiplying this by 2 will be the number of duplicate tokens that had been cleared overall.